runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/CompositeParsers/DelegatorInvokesDelegateRuleWithArgs.stg

```diff
@@ -28,5 +28,5 @@ WS : (' '|'\n') -> skip ;
 
 slaveGrammar(grammarName) ::= <<
 parser grammar S;
-a[int x] returns [int y] : B {<write("\"S.a\"")> $y=1000;} ;
```

Comment:
@ericvergnaud Notice that my proposed form is two consecutive, independent actions. I would expect the Python target to emit them each on their own line, and thus not need the separating `;`.